lib/model/frames/entity.js

```diff
@@ -37,6 +38,20 @@ class Entity extends Frame.define(
         });
       });
   }
+
+  static fromJson(body, properties, dataset) {
+    return parseCreateJson(body, properties, dataset.name)
+      .then((entityData) => {
+        if (!entityData)
+          return null; // entity create was false
+        const { uuid, label, datasetName } = entityData.system;
+        const { data } = entityData;
+        return new Entity.Partial({ uuid, datasetId: dataset.id }, {
+          def: new Entity.Def({ data, label }),
```

Comment:
some places we stick the current def in `def` and other places (the user facing ones), we put it in `currentVersion`. Would like to make it more consistent, maybe get rid of this .def?